sound/soc/intel/boards/sof_sdw.c

```diff
@@ -663,9 +663,7 @@ static int sof_card_dai_links_create(struct device *dev,
 				     struct snd_soc_card *card)
 {
 	int ssp_num, sdw_be_num = 0, hdmi_num = 0, dmic_num;
-#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA_AUDIO_CODEC)
```

Comment:
minor nit on the commit message: when you wrote 'judgement' you meant 'test'.

Reply:
Thanks for pointing it out. I will refine the patch. Sorry for the delay. I missed you comments.